common/components/Forms/AddPaymentForm/PriceList/data.ts

```diff
@@ -0,0 +1,131 @@
+// export const data = [
+//   {
+//     key: '1',
+//     number: '1',
+//     description: 'Оренда приміщення',
+//     quantity: 1,
+//     unit: 'грн.',
+//     price: '3420.00',
+//     total: '3420.00',
+//   },
+//   {
+//     key: '2',
+//     number: '2',
+//     description: 'Послуги Інтернет за місяць',
+//     quantity: 1,
+//     unit: 'грн.',
+//     price: '250.00',
+//     total: '250.00',
+//   },
+//   {
+//     key: '3',
+//     number: '3',
+//     description: 'Електроенергія',
+//     quantity: 1,
+//     unit: 'грн.',
+//     price: '864.00',
+//     total: '864.00',
+//   },
+//   {
+//     key: '4',
+//     number: '4',
+//     description: 'Опалення',
+//     quantity: 1,
+//     unit: 'грн.',
+//     price: '2064.00',
+//     total: '2064.00',
+//   },
+//   {
+//     key: '5',
+//     number: '',
+//     description: '',
+//     quantity: '',
+//     unit: '',
+//     price: '',
+//     total: '',
+//   },
+//   {
+//     key: '6',
+//     number: '',
+//     description: 'Всього:',
+//     quantity: '',
+//     unit: '',
+//     price: '',
+//     total: '6598.00',
+//   },
+// ];
+
+
+import {IDomain} from "@common/modules/models/Domain";
+import {IStreet} from "@common/modules/models/Street";
+import {IRealestate} from "@common/api/realestateApi/realestate.api.types";
+import {IService} from "@common/api/serviceApi/service.api.types";
+
+export const data = [{
```

Comment:
This is test data, right? So rename it to something more self explainable. Or, if it`s unused - remove it